Sources/CGFloatExtensions.swift

```diff
@@ -28,7 +28,7 @@ extension CGFloat {
     }
 
     /// EZSE: Converts angle degrees to radians.
-    public static func degreesToRadians(angle: CGFloat) -> CGFloat {
+    public static func degreesToRadians(_ angle: CGFloat) -> CGFloat {
```

Comment:
Why not make static? And I think degreetoradians is appropriate here because it degrees is only one of many possible units. and cgfloat doesn't mean its certainly degrees.